apps/ech.c

```diff
@@ -0,0 +1,378 @@
+/*
```

Comment:
The list app should also be educated about ECH for it's disabled listing.

Reply:
Sorry, now I look at this, I don't get the comment. Can you elaborate?

Reply:
Something needs to be added to `list_disabled()`

https://github.com/openssl/openssl/blob/1c4f9684696bad3a602b388a414f2051f0365b3d/apps/list.c#L1297-L1450

Reply:
ah, gotcha - sure, did that